### PR TITLE
PMM-7345 Expose metrics for all available databases in PG server

### PR DESCRIPTION
**Pull request by adivinho**
[PMM-7345](https://jira.percona.com/browse/PMM-7345)

- [x] Build: https://github.com/Percona-Lab/pmm-submodules/pull/1612

- [ ] Links to other linked pull requests (optional).
---
- [ ] Tests passed.
- [ ] Feature build pass.
- [ ] (Re)requested review.
- [ ] Fix conflicts with target branch.
- [ ] Update API dependency.
- [ ] Update jira ticket description if needed.
- [ ] Attach screenshots/console output to confirm new behavior to jira ticket, if applicable.

Screenshot_20210316_191832

### CODE REVIEW

File: services/agents/postgresql.go [hunk 1]
```diff
@@ -47,6 +47,8 @@ func postgresExporterConfig(service *models.Service, exporter *models.Agent, red
 		"--collect.custom_query.mr.directory=/usr/local/percona/pmm2/collectors/custom-queries/postgresql/medium-resolution",
 		"--collect.custom_query.hr.directory=/usr/local/percona/pmm2/collectors/custom-queries/postgresql/high-resolution",
 
+		"--auto-discover-databases",
+		"--exclude-databases=template0,template1,postgres",
```
Comment (BupycHuk) [inline services/agents/postgresql.go:51]: do we really should exclude template0, template1 and postgres?

**Review by BupycHuk — CHANGES_REQUESTED** [review]
Tests are failing

File: services/agents/postgresql.go (hunk 1 above)
Comment (adivinho) [reply services/agents/postgresql.go:51]: Yes we do. Please check my comment in the ticket.

Also here are some snippets from postgresql documentation: 
_"CREATE DATABASE actually works by copying an existing database. By default, it copies the standard system database named template1. Thus that database is the "template" from which new databases are made. 
...
There is a second standard system database named template0. This database contains the same data as the initial contents of template1, that is, only the standard objects predefined by your version of PostgreSQL."_

Comment (BupycHuk) [reply services/agents/postgresql.go:51]: ok, then. let's fix tests (only the one related to this ticket) and go on

Comment (adivinho) [reply services/agents/postgresql.go:51]: Done

**Review by BupycHuk — APPROVED** [review]

**Pull request merged**